bin/tx-prover/src/proxy/mod.rs

```diff
@@ -535,6 +551,135 @@ impl ProxyHttp for LoadBalancer {
             self.0.add_available_worker(worker).await;
         }
     }
+
+    // The following methods are a copy of the default implementation defined in the trait, but
+    // with tracing instrumentation.
+    // ============================================================================================
```

Comment:
I think for default implementations, we should just be able to use `ProxyHttp::{method_name}` instead of copying them (unless there's something I'm missing).

Reply:
Oh, I did not know that using ProxyHttp::{method} calls the default implementation. That is so much better. Thanks!

Reply:
Looks like I was having hallucinations and this is in fact not possible